api/v1beta1/kuadrant_types.go

```diff
@@ -54,6 +54,11 @@ func (p *Kuadrant) GetLocator() string {
 
 // KuadrantSpec defines the desired state of Kuadrant
 type KuadrantSpec struct {
+	Observability Observability `json:"observability,omitempty"`
```

Comment:
I would make this pointer, so you can discriminate between 
a) not set
b) set to false
c) set to true

Reply:
Understood.
From the reconcile function point of view, I'm wondering if there would be a difference between a and b. 
I think the behaviour would be the same for both i.e. remove the SerivceMonitors/PodMonitors if they exist.